README.md

```diff
@@ -1,2 +1,63 @@
 Symfony Messenger Polyfill
 ==========================
+
+Allows to use Symfony Messenger with Symfony 3.4 and 4.0.
```

Comment:
I thought that we are targetting support of Messenger 4.2 with Symfony 3.4 and 4.1 only as 4.0 is EOF. Am I mistaken?

Reply:
Ok, we should likely decrease in `composer.json` the min version for symfony components to `^ 3.4 || ^4.0` instead of current `^3.4 || ^4.1`.

Reply:
didn't even see that, yep, makes sense